src/general_manager/measurement/measurementField.py

```
@@ -1,168 +1,275 @@
+            max_digits=30, decimal_places=10, db_index=True, editable=editable, **nb
         )
-        self.unit_field: models.CharField[str] = models.CharField(
-            max_length=30, **null_blank_kwargs, editable=editable
-        )
-        super().__init__(null=null, blank=blank, *args, **kwargs)
+        self.unit_field = models.CharField(max_length=30, editable=editable, **nb)
 
-    def contribute_to_class(
-        self, cls: type, name: str, private_only: bool = False, **kwargs: Any
-    ) -> None:
+        super().__init__(null=null, blank=blank, editable=editable, *args, **kwargs)
```

Ruff (0.12.2)

Star-arg unpacking after a keyword argument is strongly discouraged

(B026)